components/multimedia/timidity++/Makefile

```diff
@@ -34,6 +36,9 @@ COMPONENT_LICENSE_FILE=	COPYING
 TEST_TARGET= $(NO_TESTS)
 include $(WS_MAKE_RULES)/common.mk
 
+CFLAGS += -DSTDC_HEADERS
```

Comment:
Please add a comment explaining why is this needed.  Thanks.

Reply:
Hmmm, this sounds like a bug either in illumos or in timidity++ and so it should be reported.  Please do so.